models/classes/runner/QtiRunnerService.php

```diff
@@ -295,7 +295,7 @@ public function init(RunnerServiceContext $context)
             $session->resume();
         }
 
-        $session->initItemTimer();
+        $session->startItemTimer();
```

Comment:
`init` method should not start the item timer, `getItem` will do that

Reply:
It looks like `startItemTimer()` and `initItemTimer()` are behaving the same.
https://github.com/oat-sa/extension-tao-testqti/blob/develop/models/classes/runner/session/TestSession.php#L215:L222

Reply:
Oops, this needs to be reverted. With the last change, both methods are not behaving the same.
Actually, the `startItemTimer()` will behave the same as the previous version of `initItemTimer()`. At the end, this PR does not change so much the existing behavior.